src/main/typescript/node_modules/@atomist/rug/operations/Handlers.ts

```diff
@@ -82,12 +82,12 @@ Messages got to the bot.
 Rugs are run straight away
 */
 
-type Plannable = Message | Edit | Generate | Review | Execute | Command
+type Plannable = Edit | Generate | Review | Execute | Command
```

Comment:
What is Plannable vs InstructionKind conceptually. I understand these are objects rather than strings. Why no Respond here like the InstructionKind?